src/flicking.js

```diff
@@ -282,24 +282,27 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		_setPadding: function(padding, build) {
 			var horizontal = this.options.horizontal;
 			var panel = this._conf.panel;
+			var paddingSum = padding[0] + padding[1];
+			var cssValue = {};
 
 			panel.size = this.$wrapper[
-				horizontal ? "outerWidth" : "height"
-			]() - (padding[0] + padding[1]);
+					horizontal ? "innerWidth" : "innerHeight"
+				]() - paddingSum;
 
-			var cssValue = {
-				padding: (horizontal ?
-				"0 " + padding.reverse().join("px 0 ") :
-				padding.join("px 0 ")) + "px"
-			};
+			if (paddingSum || !build) {
+				cssValue.padding = (horizontal ?
+						"0 " + padding.reverse().join("px 0 ") :
+						padding.join("px 0 ")) + "px";
+
+				cssValue[ horizontal ? "width" : "height" ] = panel.size;

```

Comment:
`box-sizing` is supported from IE8. If we given up IE7, it can be considered.
http://caniuse.com/#feat=css3-boxsizing